tox.ini

```diff
@@ -17,17 +17,16 @@ passenv = CI TRAVIS TRAVIS_*
 deps =
   flake8
 commands =
-  flake8
+  flake8 ./stix2matcher
 
-[testenv:isort-check]
+[isort]
```

Comment:
I think this should be left as it was. `[isort]` means this section holds the configuration for the isort command. `[testenv:isort-check]` means this section holds configuration for the `isort-check` test environment in tox. I checked the Travis build and the isort command was never run.

Other than that these changes look good! 👍

Reply:
@clenk I get an error with this `[testenv:isort-check]` -
```
isort-check run-test: commands[0] | isort stix2matcher --df
Traceback (most recent call last):
  File "/cti-pattern-matcher/.tox/isort-check/bin/isort", line 8, in <module>
    sys.exit(main())
  File "/cti-pattern-matcher/.tox/isort-check/lib/python3.6/site-packages/isort/main.py", line 728, in main
    config = Config(**config_dict)
  File "/cti-pattern-matcher/.tox/isort-check/lib/python3.6/site-packages/isort/settings.py", line 344, in __init__
    super().__init__(sources=tuple(sources), **combined_config)  # type: ignore
TypeError: __init__() got an unexpected keyword argument 'check'
ERROR: InvocationError for command /cti-pattern-matcher/.tox/isort-check/bin/isort stix2matcher --df (exited with code 1)
```
`ERROR:   isort-check: commands failed`

I thought replacing that with `[isort]` resolved those error and I found this status as well: `isort-check: commands succeeded`

I wonder if this is something to do with my local workspace issue.

Reply:
So I reverted back to `[isort-check]` but travis build failed on python 3.6 env throwing the same error as I posted above.

Reply:
I think it'll work if you remove these 2 lines from `isort.cfg`:
```
check=1
diff=1
```

Reply:
@clenk fixed the isort checking but looks like `python-import-sorter` is failing the build. is it necessary to have  python-import-sorter hook in pre-commit: https://github.com/oasis-open/cti-pattern-validator/blob/2dafa44d3ea37faefaaa92e703f2ac481485de31/.pre-commit-config.yaml#L15

correct me if I'm wrong it looks like isort is conflicting with ` python-import-sorter` while sorting the import in travis build. is there any args that we can use in pre-commit config file to resolve the conflict

Reply:
So removing `python-import-sorter` hook from pre-commit config resolves the error.
```
repo: https://github.com/FalconSocial/pre-commit-python-sorter
     sha: b57843b0b874df1d16eb0bef00b868792cb245c2
     hooks:
     -   id: python-import-sorter
```

Let me know if this is ok to remove.

Reply:
Looks like the problem was caused because we added pytest as a dependency but forgot to add it to isort's list of known third party libraries, which it uses to determine sort order.

Reply:
Thanks @clenk